src/library_idbfs.js

```diff
@@ -21,10 +21,8 @@ addToLibrary({
     },
     DB_VERSION: 21,
     DB_STORE_NAME: 'FILE_DATA',
-    mount: function(mount) {
-      // reuse all of the core MEMFS functionality
-      return MEMFS.mount.apply(null, arguments);
-    },
+    // reuse all of the core MEMFS functionality
+    mount: (...args) => MEMFS.mount.apply(null, ...args);
```

Comment:
This is wrong - you are now spreading `args` in the `apply` function, which means `apply` will try to interpret `args[0]` as an array and will fail at runtime.

TBH this function should be just `(mount) => MEMFS.mount(mount)` but if there might be really >1 args, then `(...args) => MEMFS.mount(...args)` is the right way to write this.